### PR TITLE
Example on how to use FindByInventoryPath

### PR DESCRIPTION
**Pull request by jm66**
This example shows how to find a VM by its inventory path. It requires the datacenter and creates a prefix to append it to the given path.

### CODE REVIEW

File: samples/find_by_inv_path.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# VMware vSphere Python SDK
+# Copyright (c) 2008-2013 VMware, Inc. All Rights Reserved.
```
Comment (michaelrice) [inline samples/find_by_inv_path.py]: might want to bump the dates here too..

File: samples/find_by_inv_path.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                          port=args.port)
+
+# doing this means you don't need to remember to disconnect your script/objects
+atexit.register(connect.Disconnect, si)
+
+# see:
+# http://pubs.vmware.com/vsphere-55/topic/com.vmware.wssdk.apiref.doc/vim.ServiceInstanceContent.html
+# http://pubs.vmware.com/vsphere-55/topic/com.vmware.wssdk.apiref.doc/vim.SearchIndex.html
+search_index = si.content.searchIndex
+inv_path_prefix = "%s/vm/" % args.datacenter
```
Comment (michaelrice) [inline samples/find_by_inv_path.py]: any reason you use the ```%s``` syntax here but below use the newer ```.format``` syntax? Personally I would be in favor of picking one or the other, with my pref being for the ```.format```

**Comment by michaelrice** [comment]
I would also suggest a from future import for print_function since you are using ```print()```

**Comment by hartsock** [comment]
Chock full of `pep8` errors!

Just clean 'em up before we go further. Thanks!

https://travis-ci.org/vmware/pyvmomi-community-samples/jobs/52436411

**Pull request closed without merging**